monailabel/tools/validate_tta.py

```diff
@@ -16,88 +16,96 @@
 
 import requests
```

Comment:
not sure if this file should be here... this doesn't seem related to the items in the PR.

Reply:
docstring warning... direct exit issue..